conan/tools/cmake/toolchain/blocks.py

```diff
@@ -502,6 +505,38 @@ class FindFiles(Block):
         {% endif %}
     """)
 
+    @staticmethod
+    def _multiconfig_generator(prop):
+        return ''.join(f'"$<$<CONFIG:{c}>:{i}>"' for c, v in prop.items() for i in v)
```

Comment:
```suggestion
        return ' '.join(f'"$<$<CONFIG:{c}>:{i}>"' for c, v in prop.items() for i in v)
```

fixing this as it need to be a space (`' '.join()`) otherwise we get this:

```
-- Using Conan toolchain: C:/Users/luisc/dev/jcar87/conan-cmake-install-runtime-dependencies/build/generators/conan_toolchain.cmake
-- Conan toolchain: CMAKE_GENERATOR_TOOLSET=v143
-- Conan toolchain: C++ Standard 14 with extensions OFF
CMake Warning (dev) at build/generators/conan_toolchain.cmake:103:
  Syntax Warning in cmake code at column 111

  Argument not separated from preceding token by whitespace.
Call Stack (most recent call first):
  build/CMakeFiles/3.29.2/CMakeSystem.cmake:6 (include)
  CMakeLists.txt:3 (project)
This warning is for project developers.  Use -Wno-dev to suppress it.
```